project_access_tokens_test.go

```diff
@@ -48,22 +48,24 @@ func TestListProjectAccessTokens(t *testing.T) {
 
 	want := []*ProjectAccessToken{
 		{
-			ID:        1876,
-			UserID:    2453,
-			Name:      "token 10",
-			Scopes:    []string{"api", "read_api", "read_repository", "write_repository"},
-			CreatedAt: &time1,
-			Active:    true,
-			Revoked:   false,
+			ID:          1876,
+			UserID:      2453,
+			Name:        "token 10",
+			Scopes:      []string{"api", "read_api", "read_repository", "write_repository"},
+			CreatedAt:   &time1,
+			Active:      true,
+			Revoked:     false,
+			AccessLevel: 40,
```

Comment:
This, and the others in this test file, need to be `AccessLevelValue`'s as well...